### PR TITLE
Adds config to timeout bolt discovery

### PR DESCRIPTION
**Pull request by DanielGarcia-Carrillo**
Sometimes all my bolts are not connected, so the discovery will just
hang and never let homebridge start the server.

### CODE REVIEW

File: lib/platform.js [hunk 1]
```diff
@@ -32,6 +42,7 @@ MisfitBoltPlatform.prototype.accessories = function(callback) {
     accessory.initialize(function() {
       if (totalAccessories == configs.length) {
```
Comment (flochtililoch) [inline lib/platform.js]: we should probably also check for `done !== true` to prevent running the callback twice.

Comment (DanielGarcia-Carrillo) [inline lib/platform.js]: Woops, completely missed that.

**Comment by flochtililoch** [comment]
Looks good, thanks for improving this!

**Comment by flochtililoch** [comment]
`v0.0.12` released on npm

**Pull request merged**